scripts/variantstore/scripts/close_bed_file_gaps.py

```diff
@@ -8,9 +8,11 @@
 argParser.add_argument("-s", "--bin-size", help="size of the bins in kb", type=int)
 argParser.add_argument("-i", "--infile", help="bed input file", type=str)
 argParser.add_argument("-o", "--outfile", help="bed output file", type=str)
+argParser.add_argument("-v", "--verbose", help="whether to be verbose in output", type=bool, default=False)
 argParser.add_argument("-l", "--interval-list", help="an interval list we can use to fill in any data on a contig past the end of our samples", type=str)
 
 args = argParser.parse_args()
+print("args=%s" % args)
```

Comment:
Is this debugging code?

Reply:
yes.  There was a lot of output from this script the last time I ran it, so I modified it to be a bit quieter by default.  This was done in preparation for bundling it into a WDL that runs automatically, and not flooding the logs with unnecessary outputs